### PR TITLE
Initial support helm deployment.

### PR DESCRIPTION
**Pull request by matrixji**
- add helm chart for nebula-studio.

Resolve issue: #20

Signed-off-by: Ji Bin <matrixji@live.com>

### CODE REVIEW

File: deployment/helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "v3"
```
Comment (MegaByte875) [inline deployment/helm/Chart.yaml]: studio latest version is v3.0.0

File: deployment/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| image.nebulaStudio.repository  |  The repository for nebula graph studio's image  | vesoft/nebula-graph-studio |
+| image.nginx.repository  |  The repository for nginx's image  | nginx  |
+| image.httpGateway.tag  |  The tag for http grateway's image  | v2  |
+| image.nebulaImporter.tag  |  The tag for nebula-importer ateway's image  | v2  |
+| image.nebulaStudio.tag  |  The tag for nebula graph studio's image  | v3  |
+| image.nginx.tag  |  The tag for nginx's image  | alpine  |
+| service.type  | The service type, should be one of ['NodePort', 'ClusterIP', 'LoadBalancer'] |  ClusterIP  |
+| service.port  | The expose port for nebula-graph-studio's web |  7001  |
+| resources.httpGateway.limits.cpu  | The cpu limit for http gateway container  | 2  |
+| resources.httpGateway.limits.memory  | The memory limit for http gateway container  | 4Gi  |
```
Comment (MegaByte875) [inline deployment/helm/README.md]: resource limit 2c4g is not needed,  what do you think?

File: deployment/helm/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+| resources.httpGateway.requests.cpu  | The cpu request for http gateway container  | 100mi  |
+| resources.httpGateway.requests.memory  | The memory request for http gateway container  | 256Mi | 
+| resources.nebulaImporter.limits.cpu  | The cpu limit for nebula importer container  | 2  |
+| resources.nebulaImporter.limits.memory  | The memory limit for nebula importer container  | 4Gi  |
```
Comment (MegaByte875) [inline deployment/helm/README.md]: ditto

File: deployment/helm/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+| resources.nebulaImporter.requests.cpu  | The cpu request for nebula importer container  | 100mi  |
+| resources.nebulaImporter.requests.memory  | The memory request for nebula importer container  | 256Mi | 
+| resources.nebulaStudio.limits.cpu  | The cpu limit for nebula studio container  | 2  |
+| resources.nebulaStudio.limits.memory  | The memory limit for nebula studio container  | 4Gi  |
```
Comment (MegaByte875) [inline deployment/helm/README.md]: ditto

File: deployment/helm/values.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  port: 7001
+
+resources:
+  httpGateway:
+    limits:
+      cpu: 2
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 256Mi
```
Comment (MegaByte875) [inline deployment/helm/values.yaml]: We usually recommend not to specify default resources and to leave this as a conscious
choice for the user. If set here, we'd better specify a lower value.

File: deployment/helm/Chart.yaml (hunk 1 above)
Comment (matrixji) [reply deployment/helm/Chart.yaml]: Agree with it.

File: deployment/helm/values.yaml (hunk 5 above)
Comment (matrixji) [reply deployment/helm/values.yaml]: For these resource limits/requests, I prefer to leave them empty to let the user define them.

Comment (wey-gu) [reply deployment/helm/values.yaml]: I strongly agree to have them empty, I will consume this masterpiece in nebula-KinD(https://github.com/wey-gu/nebula-operator-kind) and it's expected to be running even on extreme poor resource single VM(thus I had to remove all requests for all dependent resources, and by removing all requirements, it enabled nebula being playable on KataCoda: https://katacoda.com/wey/scenarios/siwi-kgqa )

**Review by MegaByte875 — APPROVED** [review]
LGTM

**Review by hetao92 — APPROVED** [review]
LGTM

**Pull request merged**